liferay-content-management/liferay-portal
# LPD-18397 Remove FF for Identify at a glance if a Document is visible for guests

### What is this trying to solve?
https://liferay.atlassian.net/browse/LPD-26299

Delete release feature Flag LPD-16311
 
### How am I fixing it?

Deleting the feature flag

### How can you verify that it works?

1. When in the Admin View for D&M, a permissions icon will be visible for the content that doesn't have Guest View permissions.
2. The permissions icon will be placed next to the Document title in the card, list, and table views.
3. The permissions icon will be placed next to the Document title in the Editor Screen.
4. In the Item Selector, we also indicate the content that doesn't have Guest View permissions with the permissions icon. Card, List, and Table views.
5. When hovering over the permissions icon a tooltip appears with "Not Visible to Guest Users" text on it.
